lxl-web/src/routes/(app)/[[lang=lang]]/find/+page.svelte

```diff
@@ -2,18 +2,55 @@
 	import SeachMapping from './SeachMapping.svelte';
 	import FacetSidebar from './FacetSidebar.svelte';
 	import SearchCard from './SearchCard.svelte';
-	export let data;
+	import { page } from '$app/stores';
+	import { goto } from '$app/navigation';
+
+	$: numHits = $page.data.searchResult.totalItems;
+	const sortOrder = $page.url.searchParams.get('_sort');
+	const sortOptions = [
+		{ value: '', label: 'Relevans' },
+		{ value: `_sortKeyByLang.${$page.data.locale}`, label: 'A-Ö' },
+		{ value: `-_sortKeyByLang.${$page.data.locale}`, label: 'Ö-A' }
+	];
+
+	function handleSortChange(e: Event) {
+		const value = (e.target as HTMLSelectElement).value;
+		let searchParams = $page.url.searchParams;
+		searchParams.set('_sort', value);
+		goto(`find?${searchParams.toString()}`, { invalidateAll: true });
+	}
 </script>
 
-<SeachMapping mapping={data.searchResult.mapping} />
+<SeachMapping mapping={$page.data.searchResult.mapping} />
 <div class="container-fluid">
 	<div class="flex gap-16 py-4 sm:py-8">
 		<div class="hidden w-80 shrink-0 md:flex">
-			<FacetSidebar facets={data.searchResult.facetGroups} />
+			<FacetSidebar facets={$page.data.searchResult.facetGroups} />
 		</div>
-		<main class="w-full">
+		<main class="max-w-content">
+			<div class="mb-4 flex justify-between">
+				<p role="status">
+					{#if numHits && numHits > 0}
+						{numHits.toLocaleString($page.data.locale)} träffar
+					{:else}
+						Inga träffar
+					{/if}
+				</p>
+				{#if numHits > 0}
+					<div>
+						<label for="search-sort">Sortera efter</label>
+						<select id="search-sort" on:change={handleSortChange}>
```

Comment:
We can add a form attribute here (e.g. `form="main-search"`) to associate the select input with the form used for searching (which enables some sort of fallback if javascript isn't available).

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select#form

Reply:
Yep. I guess it will then not react to any change in the select menu, but rather include the value on form submission (= search click). But it's better than nothing!